vscode/test/benchmark/evaluate-test-case.ts

```diff
@@ -0,0 +1,29 @@
+import { exec as _exec } from 'child_process'
+import { promisify } from 'util'
+
+import { assertEnv } from './utils'
+
+const exec = promisify(_exec)
+
+export enum CaseStatus {
+    'PASS',
+    'FAIL',
+    'NO_CHANGE',
+}
+
+export const testCompletionResult = async (
+    testFile: string,
+    testCommand: string,
+    cwd: string
+): Promise<CaseStatus.PASS | CaseStatus.FAIL> => {
+    const benchmarkDockerImage = assertEnv('BENCHMARK_DOCKER_IMAGE')
+    const dockerCommand = `docker run --mount src="${cwd}",target=/app,type=bind ${benchmarkDockerImage}`
+    let status: CaseStatus
+    try {
+        await exec(`${dockerCommand} ${testCommand} ${testFile}`, { cwd })
```

Comment:
```suggestion
        await exec(`${dockerCommand} ${testCommand}`, { cwd })
```

It might be better to leave the `testCommand` fully up to the config.json file. Some languages/tools won't want the test file itself as an argument like this

Reply:
@umpox it seems like the command output here gets eaten.

e.g. when writing a new test case, I see just:

```
Exit code:   0
Done
```

I have to go into the tmp directory and run the test command to see why it failed (compiler error output was sent to stderr):

```
# command-line-arguments
./main.go:6:5: f declared and not used
./main.go:6:8: err declared and not used
```

Maybe we can use `docker run -it` flag and capture it in the TypeScript side?

Reply:
Added logs on error!